### PR TITLE
ask-cli 1.1.2 (new formula)

### PR DESCRIPTION
**Pull request by mnin**
ask-cli is the official command line interface for Amazon Skill Kit (ASK).

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ask-cli.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"ask", "--help"
```
Comment (fxcoudert) [inline Formula/ask-cli.rb]: Can we have a test that does something more? Like tries to connect without credentials and match the error message.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/ask-cli.rb (hunk 1 above)
Comment (mnin) [reply Formula/ask-cli.rb]: @fxcoudert sure, I add a better test for this formula

**Review by fxcoudert — APPROVED** [review]

**Comment by mnin** [comment]
@fxcoudert is anything missing here?

**Comment by fxcoudert** [comment]
Thanks @mnin for the pull request!

**Pull request closed without merging**